src/components/settings.jsx

```diff
@@ -0,0 +1,188 @@
+import React, { Component } from 'react';
+import 'bootstrap/dist/css/bootstrap.css';
+import firebase from "firebase";
+import '../App.css'
+import Resizer from 'react-image-file-resizer'
+
+import {storage} from "../firebaseConfig";
+
+class Settings extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            username: "",
+            fName: '',
+            lName: '',
+            email: '',
+            bio: '',
+            signedIn: '',
+            url: '',
+            image: '', 
+            
+           
+        };
+        this.handleChange = this.handleChange.bind(this)
+    }
+    handleChange = e => {
+        let image = ''
```

Comment:
i dont think u need image variable anymore. i dont see u using it other than to console log it